src/operator-manual/tracing/01-quickstart.md

```diff
@@ -0,0 +1,418 @@
+    ▪ Using image k8s.gcr.io/ingress-nginx/kube-webhook-certgen:v1.0
+🔎  Verifying ingress addon...
+🌟  The 'ingress' addon is enabled
+```
+
+## Install Jaeger
+
+We are going to use the [Jaeger Operator](https://github.com/jaegertracing/jaeger-operator)
+to manage all the different Jaeger components.
+
+The operator can be installed in many ways, we are going to use
+its helm chart.
+
+As a first step, we need to add the helm repository containing the Jaeger Operator
+charts:
+
+```console
+helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+```
+
+Then we install the operator inside of a dedicated Namespace called `jaeger`:
+
+```console
+helm install --namespace jaeger --create-namespace jaeger-operator jaegertracing/jaeger-operator
+```
+
+This will produce an output similar to the following one:
+
+```console
+helm install --namespace jaeger --create-namespace jaeger-operator jaegertracing/jaeger-operator
+manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
+NAME: jaeger-operator
+LAST DEPLOYED: Tue Sep 28 14:54:02 2021
+NAMESPACE: jaeger
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+jaeger-operator is installed.
+
+
+Check the jaeger-operator logs
+  export POD=$(kubectl get pods -l app.kubernetes.io/instance=jaeger-operator -lapp.kubernetes.io/name=jaeger-operator --namespace jaeger --output name)
+  kubectl logs $POD --namespace=jaeger
+```
+
+Given this is a testing environment, we will use default 
+["AllInOne"](https://www.jaegertracing.io/docs/1.26/operator/#allinone-default-strategy)
+strategy. As stated on the upstream documentation: this deployment strategy is
+meant to be used only for development, testing and demo purposes.
+
+> **Note well:** the operator can deploy Jaeger in many different ways. We strongly recommend
+> to read its [official documentation](https://www.jaegertracing.io/docs/1.26/operator/).
+
+Let's create a Jaeger resource:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: all-in-one
+  namespace: jaeger
+spec:
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+EOF
+```
+
+Once all the resources have been created by the Jaeger operator, the
+Jaeger Query UI will be reachable at the following address:
+
+```console
+echo http://`minikube ip`
+```
+## Install OpenTelemetry
+
+We are going to use the [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator)
+to manage the automatic injection of the OpenTelemetry Collector sidecar
+inside of the PolicyServer Deployment.
+
+The OpenTelemetry Operator requires [cert-manager](https://cert-manager.io/docs/installation/)
+to be installed inside of the cluster.
+
+This can be done with this command:
+
+```console
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
+```
+
+Once cert-manager is up and running, the operator can be installed in this way:
```

Comment:
I think we can add also a command to make sure it's running `kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all`.

Reply:
Added, thanks